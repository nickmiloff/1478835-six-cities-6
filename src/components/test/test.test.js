import React from 'react';
import {render} from '@testing-library/react';

import Test, {TestHistory, TestStore} from './test';

describe(`Test component test`, () => {
  it(`'Test component' should render correctly`, () => {
    const {container} = render(<Test><div /></Test>);

    expect(container).toMatchSnapshot();
  });

  it(`'TestHistory component' should render correctly`, () => {
    const {container} = render(<TestHistory><div /></TestHistory>);

    expect(container).toMatchSnapshot();
  });

  it(`'TestStore component' should render correctly`, () => {
    const {container} = render(<TestStore><div /></TestStore>);

    expect(container).toMatchSnapshot();
  });
});
