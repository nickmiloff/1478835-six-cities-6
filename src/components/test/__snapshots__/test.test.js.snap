// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component test 'Test component' should render correctly 1`] = `
<div>
  <div />
</div>
`;

exports[`Test component test 'TestHistory component' should render correctly 1`] = `
<div>
  <div />
</div>
`;

exports[`Test component test 'TestStore component' should render correctly 1`] = `
<div>
  <div />
</div>
`;
